fly/commands/trigger_job.go

```diff
@@ -30,11 +32,17 @@ func (command *TriggerJobCommand) Execute(args []string) error {
 		return err
 	}
 
-	build, err := target.Team().CreateJobBuild(pipelineName, jobName)
+	var build atc.Build
+	team := command.TeamFlag.GetTeamTarget(target)
+	build, err = team.CreateJobBuild(pipelineName, jobName)
 	if err != nil {
+		if command.Team == "" {
+			fmt.Println("hint: are you missing '--team' to specify the team for the build?")
```

Comment:
this seems like it could be confusing to print on any error; the others ain't so bad since it's an explicit 404